### PR TITLE
chore: update broken links to lodestar docs

### PR DESCRIPTION
**Pull request by krauspt**
**Motivation**

I've found that many of links inside several `README.md` files are broken.

**Description**

This PR replaced broken links with working version. Each replaced link was doublechecked to see if works correctly and also maintains the context of original link.

**Steps to test or reproduce**

No testing required.

### CODE REVIEW

**Review by philknows — DISMISSED** [review]
Thanks for helping us fix the links! Missed that as we revamped our documentation nav.

File: README.md [hunk 1]
```diff
@@ -14,12 +14,12 @@
...
-- :books: Use [Lodestar libraries](https://chainsafe.github.io/lodestar/libraries) in your next Ethereum Typescript project.
-- :globe_with_meridians: Run a beacon node on [mainnet or a public testnet](https://chainsafe.github.io/lodestar/usage/beacon-management).
-- :computer: Utilize the whole stack by [starting a local testnet](https://chainsafe.github.io/lodestar/usage/local).
+- :gear: Follow the installation method for [source install](https://chainsafe.github.io/lodestar/getting-started/installation/#build-from-source), [NPM install](https://chainsafe.github.io/lodestar/getting-started/installation/#install-from-npm-not-recommended), or [Docker install](https://chainsafe.github.io/lodestar/getting-started/installation/#docker-installation) to install Lodestar. Or use our [Lodestar Quickstart scripts](https://github.com/ChainSafe/lodestar-quickstart).
+- :books: Use [Lodestar libraries](https://chainsafe.github.io/lodestar/supporting-libraries/libraries/) in your next Ethereum Typescript project.
+- :globe_with_meridians: Run a beacon node on [mainnet or a public testnet](https://chainsafe.github.io/lodestar/beacon-management/beacon-cli/).
```
Comment (nflaig) [inline README.md]:
```suggestion
- :globe_with_meridians: Run a beacon node on [mainnet or a public testnet](https://chainsafe.github.io/lodestar/getting-started/starting-a-node/).
```

**Review by nflaig — CHANGES_REQUESTED** [review]

**Comment by krauspt** [comment]
@nflaig I've messed this one. Thanks.

**Review by nflaig — APPROVED** [review]
LGTM, thanks for the update @krauspt

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.16.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.16.0) :tada:

**Pull request merged**